src/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px 16px ;
+}
+
+.card__image {
+  padding: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  vertical-align: top;
+  text-align: left;
+  padding: 0 0 4px ;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding: 0 0 16px ;
+  text-align: left;
+}
+
+.card__stars-and-reviews {
+  display: flex;
+  justify-content: space-between;
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+  align-self: flex-end;
+}
+
+.price-line {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 24px 0 0;
+}
+
+.product-price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__buy-reference {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+  margin-top: 16px;
+}
+
+.card__buy-reference:hover {
+  background-color: #fff;
```

Comment:
It is a good practice to make all colors variable. Fix all cases please

Reply:
fix it
![image](https://user-images.githubusercontent.com/98764641/213157659-0a3ded34-dc6f-4abe-a658-6de75227e137.png)

Reply:
it is still here. write in the chat if have questions
![image](https://user-images.githubusercontent.com/98764641/214568599-fec66b27-d036-412f-99ec-cdecd940e5db.png)